src/System Application/App/Azure Storage Services Authorization/src/StorageServiceAuthorization.Codeunit.al

```diff
@@ -54,6 +57,46 @@ codeunit 9062 "Storage Service Authorization"
     end;
+#endif
+
+    /// <summary>
+    /// Creates an account SAS (Shared Access Signature) for authorizing HTTP request to Azure Storage Services.
+    /// see: https://go.microsoft.com/fwlink/?linkid=2210398
+    /// </summary>
+    /// <param name="SigningKey">The signing key to use.</param>
+    /// <param name="SignedVersion">Specifies the signed storage service version to use to authorize requests made with this account SAS. Must be set to version 2015-04-05 or later.</param>
+    /// <param name="SignedServices">Specifies the signed services accessible with the account SAS.</param>
+    /// <param name="SignedPermissions">Specifies the signed permissions for the account SAS. Permissions are only valid if they match the specified signed resource type; otherwise they are ignored.</param>
+    /// <param name="SignedExpiry">The time at which the shared access signature becomes invalid.</param>
+    /// <returns>An account SAS authorization.</returns>
+    procedure CreateAccountSAS(SigningKey: SecretText; SignedVersion: Enum "Storage Service API Version"; SignedServices: List of [Enum "SAS Service Type"]; SignedResources: List of [Enum "SAS Resource Type"]; SignedPermissions: List of [Enum "SAS Permission"]; SignedExpiry: DateTime): Interface "Storage Service Authorization"
+    var
+        StorServAuthImpl: Codeunit "Stor. Serv. Auth. Impl.";
+    begin
+        exit(StorServAuthImpl.CreateSAS(SigningKey, SignedVersion, SignedServices, SignedResources, SignedPermissions, SignedExpiry));
+    end;
+
+    /// <summary>
+    /// Creates an account SAS (Shared Access Signature) for authorizing HTTP request to Azure Storage Services.
+    /// see: https://go.microsoft.com/fwlink/?linkid=2210398
+    /// </summary>
+    /// <param name="SigningKey">The signing key to use.</param>
+    /// <param name="SignedVersion">Specifies the signed storage service version to use to authorize requests made with this account SAS. Must be set to version 2015-04-05 or later.</param>
+    /// <param name="SignedServices">Specifies the signed services accessible with the account SAS.</param>
+    /// <param name="SignedPermissions">Specifies the signed permissions for the account SAS. Permissions are only valid if they match the specified signed resource type; otherwise they are ignored.</param>
+    /// <param name="SignedExpiry">The time at which the shared access signature becomes invalid.</param>
+    /// <param name="OptionalSASParameters">See table "Stor. Serv. SAS Parameters".</param>
+    /// <returns>An account SAS authorization.</returns>
+    procedure CreateAccountSAS(SigningKey: SecretText; SignedVersion: Enum "Storage Service API Version"; SignedServices: List of [Enum "SAS Service Type"];
+                                                                    SignedResources: List of [Enum "SAS Resource Type"];
```

Comment:
I don't want to nag.. But this bothers me :D Having some variables on new lines.

Reply:
I think I managed to resolve that one :) Let's run one more time.